jumpscale/entry_points/jsctl.py

```diff
@@ -27,18 +36,20 @@ def list_all():
 
 
 @config.command()
-@click.option("--name")
+@click.argument("name")
 def get(name):
-    value = get_config()[name]
+    _, obj, prop = traverse_config(name)
+    value = obj[prop]
     click.echo(format_config_parameter(name, value))
 
 
 @config.command()
-@click.option("--name")
-@click.option("--value")
+@click.argument("name")
+@click.argument("value")
 def update(name, value):
-    config = get_config()
-    config[name] = value
+    config, obj, prop = traverse_config(name)
```

Comment:
This might raise key error, need to handle this. (if the given key is wrong)
Also, need to strip extra spaces...etc (for get too)

Reply:
In case a key error is raised, a new root element with the given key should be inserted? or just report an error to the user? or create dicts along the path in the input (like makedirs=True)?